modules/logging-loki-log-access.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: CONCEPT
+[id="logging-loki-log-access_{context}"]
+= Configuring Loki log access
+In {logging} 5.8 and later versions, the Loki Operator supports managing access to logs through cluster wide policies, or by namespace. Users who are members of any group specified in the `adminGroups` field of the LokiStack CR are considered admins. Admin users have access to all types of logs in all namespaces.
+
+
+ClusterRoles grant access to different types of logs:
+* `cluster-logging-application-view` grants permissions to read application logs.
+* `cluster-logging-infrastructure-view` grants permissions to read infrastructure logs.
+* `cluster-logging-audit-view` grants permission to read audit logs.
+
+Every authenticated user has permission to read logs in their namespaces, this is supported by the clusterrole `logging-application-logs-reader` and clusterrolebinding `logging-all-authenticated-application-logs-reader`. If users want to start using the new permission schema they should delete these resources.
+
+
+.Example admin group settings
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  tenants:
+    mode: openshift-logging # <1>
+    openshift:
+      adminGroups: # <2>
+      - cluster-admin
+      - custom-admin-group # <3>
+----
+<1> Custom admin groups are only available in this mode.
+<2> Entering an empty list `[]` value for this field disables admin groups.
+<3> Overrides the default groups (`system:cluster-admins`, `cluster-admin`, `dedicated-admin`)
+
+
+
+
+
+.Example cluster wide log access setting
+[source,yaml]
+----
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: logging-all-application-logs-reader
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-logging-application-view # <1>
+subjects: # <2>
+- kind: Group
+  name: system:authenticated
```

Comment:
Does it make sense to set the name here as a custom admin group 'custom-admin-group' on here as mentioned on the above LokiStack CR.